app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
+  def random
+    random_greeting = Message.order('RANDOM()').first
+    render json: { message: random_greeting.greeting }
+  end
```

Comment:
- [ ] Great effort working on your project, but when I try to check the random greetings by navigating to the API URL it throws an error and after checking through your code I realize you haven't defined a method index in your greetings_controller. Instead of `def random`, it should be `def index`
- Also I realized I could not get the random greetings displaying on the browser in the front-end application. 

Below is a video recording for my claim 👇

[screen-capture (3).webm](https://github.com/musangamfure/hello-rails-back-end/assets/78634841/b0b4fec9-1e61-400f-8459-012b1b9b971c)